packages/webdriverio/src/index.js

```diff
@@ -38,7 +38,7 @@ export const remote = async function (params = {}, remoteModifier) {
     instance.addCommand = (name, fn, attachToElement) => (
         origAddCommand(name, runFnInFiberContext(fn), attachToElement)
     )
-
+    
```

Comment:
```suggestion
```

Reply:
@christian-bromann I thought you meant to move it in the code but I now understand you mean in the docs. I'll change it back and fix the docs tomorrow.